packages/prismarine/src/entity/other/Item.ts

```diff
@@ -1,7 +1,8 @@
-import { Vector3 } from '@jsprismarine/math';
-import type Player from '../../Player';
+import Player from '../../Player';
 import type ContainerEntry from '../../inventory/ContainerEntry';
 import { AddItemActorPacket } from '../../network/Packets';
+import { Position } from '../../world/Position';
+import type { World } from '../../world/World';
 import { Entity } from '../Entity';
 
 export class Item extends Entity {
```

Comment:
Well, an Item is an Entity, but its runtime ID isn't correlated to the Entity one, we will see about this one.